src/plugins/vis_type_timelion/server/plugin.ts

```diff
@@ -83,9 +88,12 @@ export class Plugin {
     };
 
     functionsRoute(router, deps);
-    runRoute(router, deps);
     validateEsRoute(router);
 
+    core.getStartServices().then(([_, depsStart]) => {
```

Comment:
I can recommend you to pass `getStartServices` as a dependency into `runRoute`. Your solutions works fine, but I think it would be more correct to register the route in the correct `setup` phase

Reply:
done